apps/comps/components/navbar/index.tsx

```diff
@@ -23,9 +23,7 @@ import { config } from "@/config/public";
 import { useSession } from "@/hooks";
 import { useClaim } from "@/hooks/useClaim";
 
-import { NonStakeBoost } from "./NonStakeBoost";
 import { RecallToken } from "./RecallToken";
-import { StakeBoost } from "./StakeBoost";
```

Comment:
Let's remove these files if the components aren't used anywhere. No reason to keep unused code around and clutter up the repo with files.